exporter.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+import logging
+from openpyxl import Workbook
+import time
+
+from structlog import wrap_logger
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def create_comments_book(survey_id: str, comments) -> str:
+    logger.info(comments)
+    logger.info("Generation excel file ")
+    wb = Workbook()
+    wb.create_sheet('Exported comments')
+    row = 2
+    ws = wb.active
+    ws.cell(1, 1, "Survey ID : " + survey_id)
+    ws.cell(1, 2, "Comments found : " + str(len(comments)))
+    for survey, comment in enumerate(comments):
```

Comment:
I don't think you need the `enumerate` here. Should just work with `for comment in comments`?

Reply:
it s a list of dictionaries. I am not expert but I think you need to .

Reply:
just tested it out and looks like `for comment in comments` works fine but works either way